File: web2backend/DAL/Model/Order.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace DAL.Model
{
    public class Order
    {
        public long Id { get; set; }
        public User User { get; set; }
        public long UserId { get; set; }
        public string Comment { get; set; }
        public string Address { get; set; }
        public string City { get; set; }
        public string Zip { get; set; }
        public List<OrderItem> OrderItems { get; set;}

        public bool Shipped { get; set; }
        public bool Canceled { get; set; }
    }
}

File: web2frontend/src/Models/Order.js
import OrderItem from './OrderItem'

class Order {
    constructor(id,comment,address,city,zip) {
      this.UserId=id;
      this.Items = [];
      this.Comment=comment;
      this.Address=address;
      this.City=city;
      this.Zip=zip;
    }
    
    addOrderItem(id, amount) {
      this.Items=[...this.Items, new OrderItem(id,amount)];
    }
  }
  
  export default Order;